opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -95,14 +95,141 @@ message InstrumentationLibraryMetrics {
 //                        |+-----+                    |
 //                        +---------------------------+
 //
+// Each distinct type of DataPoint represents the output of a specific
+// aggregation function, the result of applying the DataPoint's
+// associated function of to one or more measurements.
+//
 // All DataPoint types have three common fields:
-// - Labels zero or more key-value pairs associated with the data point.
-// - StartTimeUnixNano MUST be set to the start of the interval when the data's
-//   type includes an AggregationTemporality. This field is not set otherwise.
-// - TimeUnixNano MUST be set to:
-//   - the moment when an aggregation is reported (independent of the
-//     aggregation temporality).
-//   - the instantaneous time of the event.
+// - Attributes includes key-value pairs associated with the data point
+// - TimeUnixNano is required, set to the end time of the aggregation
+// - StartTimeUnixNano is optional, but strongly encouraged for DataPoints
+//   having an AggregationTemporality field, as discussed below.
```

Comment:
Should it be required for "Delta"?

Reply:
I don't think we can force it to be required, even for delta.  We *can* enforce OTLP generated by an OpenTelemetry SDK have this.

Specifically, for StatsD, while you get deltas, you don't get a "start time" for that delta (AFAIK, maybe some extension adds this?).   Is there another way to represent that or do we do something differently today?

Reply:
I mentioned in #274 that we might use Temporality=INSTANTANEOUS to represent this case intentionally, where a delta has a time but no start time. How is that different from a raw, synchronous (UpDown)Counter instrument event? The problem 

We already say "strongly encouraged" for all points with aggregation temporality, which I think is good enough. The logic outlined below, for cumulative aggregation temporality, doesn't work for delta temporality, but it seems less important to spell out a translation from deltas-without-start-times than to spell out how to encode raw Counter events.